_posts/2015-07-15-testautomation-allergy.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: Test automation allergy
+author: pekkama
+excerpt: Questioning the snake-oil approach to test automation
+---
+
+I am allergic to test automation!
+---------------------------------
+
+That's correct! I get itchy and my eyes redden every time someome talks about test automation. I get coarse throat when I see a test automation coverage report. I get twitches when we justify our lack of skills as testers with 95% line coverage. That's a reaction to poorly executed and poorly justified test automation. I am allergic to s***ty test automation. And I think everyone should be!
+
+This post is about questioning the snake-oil approach to test automation. I have been doing software testing for a number of years in different types of organizations and workplaces. My approach has been, and still is, skill-driven and holistic. Test automation done well is worth the money invested in it. There has been only few cases where it has been worth the money, though. The best automation is always a supporting act to other testing activities, and the best automation is treated that way.
+
```

Comment:
"Test automation done well is worth the money invested in it. There has been only few cases where it has been worth the money, though." Outo lause. Tarkoitatko jotain tähän suuntaan: "Test automation done well is worth the money invested in in it. Sadly I've only seen a few cases where it has actually been done well, and thus been worth the money."?

Ei välttämättä kovin hyvä tuo mun lause, mutta lukijana tuntuu että alkuperäisessä versiossa on ristiriita, eikä tule selväksi mitä yrität sanoa.